components/twitter/user-tweets.js

```diff
@@ -1,23 +1,21 @@
 const twitter = require('https://github.com/PipedreamHQ/pipedream/components/twitter/twitter.app.js')
 const moment = require('moment')
- 
+
 module.exports = {
-  name: "user-tweets", 
+  name: "User Tweets",
+  description: "Emit new Tweets posted by a user.", 
   version: "0.0.1",
   props: { 
     db: "$.service.db",
     twitter,
-    from: { propDefinition: [twitter, "from"] }, 
```

Comment:
Are all of the removed props just not relevant for the user timeline API?

Reply:
correct